style/properties/shorthands/text.mako.rs

```diff
@@ -20,8 +21,11 @@
         context: &ParserContext,
         input: &mut Parser<'i, 't>,
     ) -> Result<Longhands, ParseError<'i>> {
-        % if engine == "gecko":
-            let (mut line, mut style, mut color, mut thickness, mut any) = (None, None, None, None, false);
+        % if engine == "gecko" or engine == "servo":    
```

Comment:
No need for this condition, it should always hold.

Reply:
done